### PR TITLE
I18n support

### PR DESCRIPTION
**Pull request by grossbart**
The goal of these changes is to make it possible to showcase non-latin-based typefaces, e.g. Arabic or Chinese. I think with this it should now be possible.

I decided to also make the sample text selected by default if one is provided as showing a latin pangram for an Arabic font doesn't make any sense …

### CODE REVIEW

File: packages/typography/src/fallback.tsx [hunk 1]
```diff
@@ -181,3 +187,9 @@ const FallbackOverlay = styled("div")`
 
 const getFontSize = ({ baseFontSize, msRatio }: Theme, level: number = 0) =>
   `${(baseFontSize / 16) * Math.pow(msRatio, level)}em`;
+
+const  activeFontStack = (cut: FontFace, activeFallback: undefined | string) => {
+  const stack = [cut.fontFamily, ...cut.fallback]
+  const stackIndex = activeFallback ? stack.indexOf(activeFallback) : 0
+  return stack.slice(stackIndex).join(',')
```
Comment (wereHamster) [inline packages/typography/src/fallback.tsx]: I suspect the font family names have to be quoted if they contain strings.

**Comment by grossbart** [comment]
Good point – also, this function is used a lot in my app, so it might make sense to make it available externally as a `FontStack -> string` helper. Yes? And where to put it?

**Comment by wereHamster** [comment]
Quotes are not needed but recommended: https://stackoverflow.com/a/13752149/1075783.

Why is such a function needed in your app?

**Comment by grossbart** [comment]
You defined this function twice in Going Digital, so that's one reason ;) My reason is that I need to serialize the fontFamily/fallback combo into the global CSS, which is not driven by Emotion.

Due to wrapping "sans-serif" being worse than not wrapping the name, I'm not sure whether we should just leave it as is or write something a bit more involved that checks for spaces and only wraps then …

**Comment by wereHamster** [comment]
It would be nice to test this new feature in the documentation page. Pick one font from [Google Fonts](https://fonts.google.com/?category=Serif) which has a space in its name (and looks sufficiently similar to Tuna), add the CSS `@import url(…)` to the globalStyles in pages/index.tsx, and then add the font into the fallback chain in packages/typography/docs/index.tsx. Then we should see if the quoting is done correctly. I'll soon add [jest](https://jestjs.io) so we can start writing unit tests for these things.

**Comment by wereHamster** [comment]
Looks good and works. Will merge and release so you have it ready on Monday.

**Pull request merged**